### DISCUSSION TITLE
When creating a config map with preserve file checkbox checked, deleted files persist

### DISCUSSION THREAD

**Discussion by rmuller-vu** [mia-platform/community#479] [General]
### Description

When we have the `Preserve files and directories already existing in the Runtime Mount Path directory` checkbox selected on ConfigMap section the deleted files still show on the payload that is sent to the Orchestrator Generator. In Fig 1 I added a file (willdelete.json) to the configmap that will be deleted later before I save. In Fig 2 I added another file (notdeleted.json) that will be kept. In Fig 3 I show that the file (willdelete.json) from Fig 1 is deleted. In Fig 4 when I save you can see the deleted file (willdelete.json) still shows in the payload. 

Fig. 1
![image](https://github.com/mia-platform/community/assets/136749338/5b417c69-8ce1-4a17-b3e5-0ed23c4b9448)

Fig. 2
![image](https://github.com/mia-platform/community/assets/136749338/48a74cbc-041b-4b27-9ea8-330024fa2306)

Fig. 3
![image](https://github.com/mia-platform/community/assets/136749338/99c12d74-79a1-4629-9d12-17a7f425455b)

Fig 4
![image](https://github.com/mia-platform/community/assets/136749338/446df569-ce77-4f38-a047-1fa7c91e154f)

**Comment by fredmaggiowski** [comment]
Hi @rmuller-vu thanks for reaching out, this seems like an unexpected behaviour: you shouldn't receive the file if you deleted it before saving the configuration.

Please contact you Mia-Platform referent in order to track the bug so that we can address it.
